### DISCUSSION TITLE
pnpm v9 support

### DISCUSSION THREAD

**Discussion by mnigh** [dependabot/dependabot-core#9852] [Q&A] [answered]
When can we expect pnpm v9 support as the [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#:~:text=npm-,v7%2C%20v8,-poetry) currently only list support for v7 and v8?

**Comment by abdulapopoola** [comment]
Tagging @thavaahariharangit as FYI

**Comment by mnigh** [comment, marked as answer]
pnpm v9 now supported according to the [docs](https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#supported-ecosystems-and-repositories)!
